src/world/area_kkj/kkj_17/AF1C60.c

```diff
@@ -13,7 +15,87 @@ static char* N(exit_str_1) = "";
 
 #include "world/common/CheckItemFlags40.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_17/AF1C60", func_802402BC_AF1F1C);
+ApiStatus func_802402BC_AF1F1C(Evt* script, s32 isInitialCall) {
+    PopupMenu *menu;
+    s32 numEntries;
+    s32 index;
+    s32 temp;
+    s32 var1;
+    s32 var2;
+    s32 i;
+
+    if (isInitialCall) {
+        menu = heap_malloc(sizeof(*menu));
+        script->functionTempPtr[2] = menu;
+        script->varTable[10] = script->varTable[0];
+
+        if (script->varTable[0] == 0) {
+            script->varTable[1] = 0xF8405D64;
```

Comment:
these are like 129999510 , etc. stuff all within GSWF range, I think

Reply:
You are right. Totally missed those because they're positive but it makes sense once you convert.